samza-core/src/main/java/org/apache/samza/config/JobConfig.java

```diff
@@ -466,4 +472,12 @@ public boolean getStartpointEnabled() {
   public boolean getContainerHeartbeatMonitorEnabled() {
     return getBoolean(CONTAINER_HEARTBEAT_MONITOR_ENABLED, CONTAINER_HEARTBEAT_MONITOR_ENABLED_DEFAULT);
   }
+
+  public boolean getElasticityEnabled() {
```

Comment:
not sure why this is needed?

Reply:
is the question why `getElasticityEnabled` is needed when we have `getElasticityFactor`?
callers wanting to know if elasticity is enabled need not and should not be required to know that elasticity.factor = 1 means disabled and >1 means enabled.